### DISCUSSION TITLE
How to remove audience noise?

### DISCUSSION THREAD

**Discussion by yudcjbvkl** [Anjok07/ultimatevocalremovergui#1385] [Q&A]
I have a live song with audience sound and I would like to have the instrumental version without it. Is there a model that can recognize it and put it in the vocal-only version or something? I have already tried multiple models but is still very audible in the final result

**Comment by flint-hub** [comment]
They released a new model called "UVR-MDX-NET Crowd HQ 1", ive used it and it works fine for me

**Reply by Pete70s** [reply]
It also works for isolating the crowd noise, if you want to use it at a lower volume.

**Comment by kabrnamartin3-ui** [comment]
Vocals remover
